File: Day Two/JavaScript Phasegate/TaskOne.js
const prompt = require("prompt-sync")();

let scores = [10] 

for(let scoreCount = 0; scoreCount < 10; scoreCount++) {
	let score = Number(prompt("Enter scores: "));
	}
	
	for(let scorePrint = 0; scorePrint < 10; scorePrint++) {
	console.log(scores[scorePrint]);
	}

File: Day Two/java Phasegate/TaskOne.java
import java.util.Arrays;
import java.util.Scanner;
public class TaskOne { 

	public static void main(String[] args) { 
	Scanner scanner = new Scanner (System.in);
	
	int[] scores = new int [10];

	for(int scoreCount = 0; scoreCount < 10; scoreCount++) {
	System.out.println("Enter scores: ");
	scores[scoreCount] = scanner.nextInt();
	}

	for(int scorePrint = 0; scorePrint < 10; scorePrint++) {
	System.out.printf(Arrays.toString(scores[scorePrint]));
	}
	}
}